.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+
+**Neos and Flow is community software built on open-source principles by a community collaborating on a shared goal. Everyone is welcome to contribute to the community and has the possibility to influence the project.**
+
+Please be considerate of our [code of conduct][1]! We like to provide a positive and enjoyable environment for everybody.
+
+Do you want to get a certain improvement in Neos or Flow ? Did you find a problem? Do you want to help translate, promote or document the products?
+
+Become part of the Neos Project Community and help us offer a better experience to the content management industry.
+
+----------
+
+# How to participate
+
+The development of the project is structured in an agile way. You can found more details about the current teams on the official Neos website. You can join one of those team or decide to simply send a single or irregular contributions. All contributions are welcome.
+Our [Code contribution guidline](https://discuss.neos.io/t/code-contribution-guideline/503) might also be helpful to start with.
+
+## Improve
+
+Your are a developer, a design or an UX expert and want to improve the products, what's the next step?
+
+## You are a design or UX expert ?
+
+The Neos project is focused on providing first class UX for our products. Help out with Neos CMS user interaction design or graphic design! Get involved on the [design team page][6]!
```

Comment:
The linked page does not exist. This is a bit of a problem because we need those people IMHO, but there simply is no team at the moment....